### PR TITLE
Tumor sample control sample variables

### PR DESCRIPTION
**Pull request by vinjana**
* Conda env-update due to lost r-purrr-0.2.2-1.
 * More documentation.
 * New feature (API to jobs), minor version bump to 1.3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,43 +34,70 @@ The name of the Conda environment is arbitrary but needs to be consistent with t
...
+| CHR_SUFFIX | "" | Suffix added to the chromosome names |
+| CHR_PREFIX | "" | Prefix added to the chromosome names |
+| extractSamplesFromOutputFiles | true | |
+| CHROMOSOME_INDICES | empty | Bash-array of chromosome names to which the analysis should be restricted |
+
+
+# Example call
+
+```bash
+
```
Comment (dankwart-de) [inline README.md]: ? If you don't fill it now, maybe make an issue and remove it here.

File: resources/analysisTools/indelCallingWorkflow/canopyCluster_updatedFunction.R [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #
 # Copyright (c) 2018 German Cancer Research Center (DKFZ).
 #
-# Distributed under the GPL-3 License (license terms are at https://github.com/DKFZ-ODCF/IndelCallingWorkflow).
+# Distributed under the GPL-2+ License (license terms are at https://github.com/DKFZ-ODCF/IndelCallingWorkflow).
```
Comment (dankwart-de) [inline resources/analysisTools/indelCallingWorkflow/canopyCluster_updatedFunction.R]: 2+ is this the used termin for this? I could not find it with a brief search.

Comment (dankwart-de) [reply resources/analysisTools/indelCallingWorkflow/canopyCluster_updatedFunction.R]: Also it is under dual license right?

File: src/de/dkfz/b080/co/indelcallingworkflow/IndelCallingWorkflow.java [hunk 3]
```diff
@@ -37,10 +38,13 @@ public boolean execute(ExecutionContext context, BasicBamFile _bamControlMerged,
 
         TumorBamFile bamTumorMerged = new TumorBamFile(_bamTumorMerged);
 
-        if (isControlWorkflow())
+        configurationValues.add(new ConfigurationValue("tumorSample", ((COFileStageSettings) bamTumorMerged.getFileStage()).getSample().getName()));
```
Comment (dankwart-de) [inline src/de/dkfz/b080/co/indelcallingworkflow/IndelCallingWorkflow.java]: There are shorter methods for this in the Workflow and BasicBamFile class.
```suggestion
        setEnv("tumorSample", bamTumorMerged.getSample().getName(), "string")
        setEnv("controlSample", bamControlMerged.getSample().getName(), "string")
```
The values are attached to the context configuration.

Comment (dankwart-de) [reply src/de/dkfz/b080/co/indelcallingworkflow/IndelCallingWorkflow.java]: If you do not like the setEnv syntax, you can also use cvalue(), which does the same.

File: src/de/dkfz/b080/co/indelcallingworkflow/IndelCallingWorkflow.java [hunk 4]
```diff
@@ -37,10 +38,13 @@ public boolean execute(ExecutionContext context, BasicBamFile _bamControlMerged,
...
+        if (isControlWorkflow()) {
+            configurationValues.add(new ConfigurationValue("controlSample", ((COFileStageSettings) _bamControlMerged.getFileStage()).getSample().getName()));
```
Comment (dankwart-de) [inline src/de/dkfz/b080/co/indelcallingworkflow/IndelCallingWorkflow.java]: Use snippet from above

File: resources/analysisTools/indelCallingWorkflow/canopyCluster_updatedFunction.R (hunk 2 above)
Comment (vinjana) [reply resources/analysisTools/indelCallingWorkflow/canopyCluster_updatedFunction.R]: You'll find it by searching in Github.

**Pull request merged**